lib/PDL/Graphics/Gnuplot.pm

```diff
@@ -5266,7 +5300,8 @@ our $cOptionsAbbrevs = _gen_abbrev_list(keys %$cOptionsTable);
 	lc => ["linecolor"],
 	pt => ["pointtype"],
 	ps => ["pointsize"],
-	fs => ["fillstyle"]
+	fs => ["fillstyle"],
+	fc => ["fillcolor"]
```

Comment:
Can you please add a `,` on the end, so one could add another line here without spurious diffs, given Perl allows that?